manifests/profiles/default.yaml

```diff
@@ -106,12 +111,15 @@ spec:
             - port: 80
               name: http2
               targetPort: 8080
+              protocol: TCP
```

Comment:
does it break if this is not there? will users with custom IOP ports break

Reply:
yes it breaks because of this issue: kubernetes-sigs/structured-merge-diff#130 but the fix would not be available before 1.20. We can add logic to set the default protocol in the code.

Reply:
added the logic to populate the default protocol before apply